yarn.lock

```diff
@@ -1,8979 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.

```

Comment:
we should not delete this file from the repo, it should be here but change when we change a dependency.